### PR TITLE
f(e2e):  setup, add test and ci job

### PR DESCRIPTION
**Pull request by timothy-wan**

### CODE REVIEW

**Review by julie-huang — APPROVED** [review]
LGTM, looking into the CI issue now

File: .circleci/config.yml [hunk 1]
```diff
@@ -57,3 +70,16 @@ workflows:
           name: build_feature
           requires:
             - checkout_code
+
+  master:
```
Comment (julie-huang) [inline .circleci/config.yml]: does this need to be main?

Comment (timothy-wan) [reply .circleci/config.yml]: oh right!

**Comment by timothy-wan** [comment]
CI failing because deployed version doesn't have the data attribute on the button

**Pull request merged**